internal/web/app/accordion-view.js

```diff
@@ -0,0 +1,421 @@
+                // Open sections with errors and add error highlighting
+                if (validation.errors.includes(VALIDATION_ERRORS.VERSION_REQUIRED)) {
+                    if (!this.openSections.includes('version')) {
+                        this.openSections.push('version');
+                    }
+                }
+                if (validation.errors.includes(VALIDATION_ERRORS.KUBERNETES_DISTRIBUTION_REQUIRED)) {
+                    if (!this.openSections.includes('kubernetes')) {
+                        this.openSections.push('kubernetes');
+                    }
+                }
+
+
+                // Focus on the first field with an error using Alpine.js refs
+                this.$nextTick(() => {
+                    setTimeout(() => {
+                        this.focusFirstErrorField(validation.errors);
+                    }, 300);
+                });
+            }
+            
+            return validation;
+        },
+
+        // Focus first error field using Alpine.js reactive patterns
+        focusFirstErrorField(errors) {
+            let targetField = null;
+
+            if (errors.includes(VALIDATION_ERRORS.BASE_IMAGE_REQUIRED)) {
+                targetField = this.$refs.baseImageField;
+            } else if (errors.includes(VALIDATION_ERRORS.VERSION_REQUIRED)) {
+                targetField = this.$refs.versionField;
+            } else if (errors.includes(VALIDATION_ERRORS.KUBERNETES_DISTRIBUTION_REQUIRED)) {
+                targetField = this.$refs.kubernetesFields?.[0];
+            }
+
+            if (targetField) {
+                targetField.focus();
+                targetField.scrollIntoView({ behavior: 'smooth', block: 'center' });
+            }
+        },
+
+        // Handle form submission through Alpine.js store communication
+        handleFormSubmit(event) {
+            event.preventDefault();
+
+            // Run validation before proceeding
+            const validation = this.validateForm();
+
+            if (!validation.isValid) {
+                // Validation failed - accordion sections should already be opened by validateForm()
+                // Visual feedback is handled by the accordion component
+                return; // Don't proceed with form submission
+            }
+
+            // Communicate with modal component through Alpine store
+            const formData = new FormData(event.target);
+            this.$store.formSubmission = { shouldSubmit: true, formData: formData };
+        },
+
+        // Accordion-specific helper methods
+        getSectionIcon(sectionName) {
+            const icons = {
+                'base-image': '📦',
+                'architecture': '🏗️',
+                'model': '🔧',
+                'variant': '⚙️',
+                'kubernetes': '☸️',
+                'kubernetes-release': '📋',
+                'version': '🏷️',
+                'configuration': '⚙️',
+                'artifacts': '📦'
```

Comment:
these two are duplicated icons

Reply:
actually, these icons are legacy code (from previous experiments). They are not used anywhere. Removed along with more unused code.